src/app/components/player/player.js

```diff
@@ -9,8 +9,17 @@ angular.module('app.components.player', [])
 
       elem.on('mousedown', function(){
         elem.parent().on('mousemove', function(e){
-          console.log(e);
-          elem.css('top', e.clientY + "px");
+          console.log(e.clientY);
+
+          if (e.clientY > 264) {
```

Comment:
Looks good. 

For fun, you should try and get rid of the if-else statements..

If you re-arrange some of the logic and use a ternary expression, you'll find there is another way to write this out.